client/signup/steps/plans/index.jsx

```diff
@@ -29,22 +28,8 @@ import './style.scss';
 export class PlansStep extends Component {
 	state = {
 		isDesktop: isDesktop(),
-		experiment: null,
-		experimentLoaded: false,
 	};
 
-	componentWillMount() {
-		if ( this.props.flowName === 'onboarding' || this.props.flowName === 'launch-site' ) {
-			loadExperimentAssignment( 'calypso_signup_monthly_plans_default_202201_v1' ).then(
-				( experiment ) => {
```

Comment:
@taggon Heads up that we don't want to remove this because it's a new experiment that was launched three days back. Check pbxNRc-1aI-p2.

Reply:
Thanks @niranjan-uma-shankar. Looks like I might accidentally delete the experiment that is restored now.